app/src/androidTest/java/org/systers/mentorship/SignUpActivityTest.kt

```diff
@@ -140,7 +139,46 @@ class SignUpActivityTest {
         onView(withId(R.id.btnSignUp)).perform(click())
 
         onView(withId(R.id.tiConfirmPassword)).check(matches(hasTextInputLayoutErrorText("Passwords didn't match!")))
+    }
+
+    /**
+     * This test method simply checks that if the password is too weak
+     * and user clicks on signUp button then the error message is coming in
+     * password editText
+     */
+    @Test
+    fun whenPassIsTooWeakShowsErrorTest() {
+        findEditTextInTextInputLayout(R.id.tiName).perform(typeText("Mohak Gupta"), closeSoftKeyboard())
+        findEditTextInTextInputLayout(R.id.tiUsername).perform(typeText("mohak1283"), closeSoftKeyboard())
+        findEditTextInTextInputLayout(R.id.tiEmail).perform(typeText("mohak1283@gmail.com"), closeSoftKeyboard())
```

Comment:
Please change this email address. I think you're using real data, right? you should use fake data, example. Can you change the user data without reference to mentors data?

Reply:
@isabelcosta I've changed that. That was already there since the PR #74 so I thought that I shouldn't be changing that in order to keep PR clean. But I totally agree with you and I used some fake data :)